src/libraries/Microsoft.PowerFx.Core/Syntax/Visitors/ListFunctionVisitor.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.PowerFx.Syntax
+{
+    internal class ListFunctionVisitor : IdentityTexlVisitor
+    {
+        // FullName --> Name 
+        // Use Fullname as key because it's unique. 
+        public Dictionary<string, string> _functionNames = new Dictionary<string, string>();
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

Instead of having a public member, declare it as private, and expose it as an IEnumerable<string> instead